### DISCUSSION TITLE
How can we verify this configuration management.http.max_body_size in rabbitmq ?

### DISCUSSION THREAD

**Discussion by sakshiag** [rabbitmq/rabbitmq-server#11084] [Questions] [answered]
We are trying to set new configuration management.http.max_body_size  according to [rabbitmq/rabbitmq-server#9708](https://github.com/rabbitmq/rabbitmq-server/pull/9708)

Erlang Verison :  26.2.3
 Rabbitmq Version : 3.13.1
 Image :UBI9

How can we verify if it is using the set value ?   Can you please help us with this?

**Comment by the-mikedavis** [comment]
That PR was released for v3.12.x in https://github.com/rabbitmq/rabbitmq-server/releases/tag/v3.12.7 (see https://github.com/rabbitmq/rabbitmq-server/pull/9709). You'll need to upgrade v3.12.7 for that limit to take effect.

**Reply by michaelklishin** [reply]
By "3.12.7" you mean "to the latest 3.12.x or 3.13.x" ;)

**Comment by michaelklishin** [comment]
You can simply try it. If a certain configuration key is not supported by the version you are running, the node will fail to start and will report that the configuration key is not known.

You [should move to 3.13.x](https://www.rabbitmq.com/release-information), by the way, in order to have community support now.

**Reply by sakshiag** [reply]
I am sorry I mentioned the version incorrect. We are trying to add this configuration for 3.13.x. i will correct it in the description also

**Comment by michaelklishin** [comment, marked as answer]
``` shell
RABBITMQ_CONFIG_FILE=./management.http.max_body_size.conf sbin/rabbitmq-server
```

works just fine for me with the following `rabbitmq.conf`:

``` ini
management.http.max_body_size = 10000000
```

The docs have a dedicated section on [how to verify effective configuration](https://www.rabbitmq.com/docs/configure#verify-configuration-effective-configuration):

``` shell
rabbitmq-diagnostics environment | grep body
# =>      {max_http_body_size,10000000},
```
